### DISCUSSION TITLE
getId in Tab Screen on version 5.x.x.

### DISCUSSION THREAD

**Discussion by mahi8813** [react-navigation/react-navigation#11392] [Q&A] [answered]
getId for a Tab.Screen is not working in 5.x.x versions. But docs says getId support is there. How to make it work in 5.x.x.
Support was added in this commit https://github.com/react-navigation/react-navigation/commit/7430c723ced1101191cfbdf0c26d0b0947602b2b

**Comment by mahi8813** [comment]
@satya164 Any suggestion on the above ?

**Reply by satya164** [reply, marked as answer]
5.x is deprecated. you should upgrade
